### DISCUSSION TITLE
Parenting a mesh to another meshes bone

### DISCUSSION THREAD

**Discussion by Unwarped** [jnsmalm/pixi3d#61] [Q&A] [answered]
I want to parent a mesh to a bone on another mesh. For example a hat to an animated character model's head/neck.
Is this supported? Thanks.

**Comment by jnsmalm** [comment, marked as answer]
Yes, it's supported. You can access the joints/bones from the skin property of the mesh. A joint is pretty much a Container3D, so you can add children to it.

```javascript
let head = model.meshes[0].skin.joints.find(joint =>
    joint.name === "name_of_head")
head.addChild(PIXI3D.Mesh3D.createCube())
```

**Comment by Unwarped** [comment]
Awesome! Thank you jnsmalm it works perfect.
